Bio/SeqIO/QualityIO.py

```diff
@@ -530,7 +534,7 @@ def phred_quality_from_solexa(solexa_quality: float) -> float:
     return 10 * log(10 ** (solexa_quality / 10.0) + 1, 10)
 
 
-def _get_phred_quality(record: SeqRecord) -> Union[list[int], list[float]]:
+def _get_phred_quality(record: SeqRecord) -> Union[list[float], list[int]]:
```

Comment:
Is there any reason for swapping the order here (int, float) versus (float, int)?

Reply:
Not really, this is actual a reversion back to the original. When I was rebasing a previous pull request I accidentally swapped the order. Somewhere in a middle commit I had been fiddling with that annotation to try to get the type checker happy, but I accidentally put it back backwards

Reply:
Oddly enough though, I think `Union[list[float], list[int]]` is equivalent to `list[float]` according to type-checkers because `int` is considered a `float` for type checkers, even though it isn’t a subclass at runtime

Reply:
Really? I prefer them both to be listed explicitly for humans to read ;)

Reply:
Yeah, I kind of agree, although it is a lot more space. I'm looking forward to 3.9 getting end-of-lifed so I can use the more compact `|` notation in my code without worrying about compatibility